packages/communication-react/review/communication-react.api.md

```diff
@@ -929,9 +933,6 @@ export const createAzureCommunicationChatAdapter: ({ endpointUrl, userId, displa
 // @public (undocumented)
 export const createAzureCommunicationChatAdapterFromClient: (chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient) => Promise<ChatAdapter>;
 
```

Comment:
createDefaultHandlers are still needed:
(Example)[https://github.com/PorterNan/ui-toolkit-quickstart-composites/blob/eacf1996b997257fc5c97768babc44793001f016/src/composites/CallComposite/hooks/useAzureCommunicationHandlers.ts]

We need it to expose the handler methods for an easier implementation of component level

Reply:
I want to discuss this one with you, but this PR is a bad place for it, so adding it back here.